rs3-chunkpicker-chunkinfo-export.json

```diff
@@ -216646,6 +216713,12 @@
 					"Access members content": "Nonskill"
 				}]
 			},
+			"The Kendal": {
```

Comment:
Kendal shouldn't be quest locked, access to the cave (``Mountain camp cave``) should be. (Wiki also mentions it's impossible to enter the cave without completing the tasks)